cmake/modules/CheckWineGcc.cmake

```diff
@@ -1,6 +1,6 @@
 INCLUDE(CheckCXXSourceCompiles)
 
-FUNCTION(CheckWineGcc BITNESS WINEGCC_EXECUTABLE RESULT)
+FUNCTION(CheckWineGcc MPLATFORM WINEGCC_EXECUTABLE RESULT)
```

Comment:
I think it would be better if you didn't modify this parameter and instead implemented the special handling for ARM64 inside this function rather than at the call site.

Reply:
Done.